docs/cri/registry.md

```diff
@@ -41,6 +41,27 @@ not specified by Kubernetes via CRI.
 
 After modifying this config, you need to restart the `containerd` service.
 
+### Configure Docker Hub Credentials Example
+
+You may want to configure Docker Hub credentials to pull images from Docker Hub.
+This allows you to pull public and private images from Docker Hub without hitting rate limits.
+
+To configure Docker Hub credentials, modify `/etc/containerd/config.toml` as follows:
+
+```toml
+[plugins."io.containerd.grpc.v1.cri".registry.configs."registry-1.docker.io".auth]
+username = "example-username"
+password = "example-password"
+```
+
+> Docker [access tokens](https://docs.docker.com/docker-hub/access-tokens/) are supported. These are passwords that start with `dckr_pat_`.
```

Comment:
can you elaborate a bit on this point...

Reply:
Docker has been around long enough that I've run into long-time users who have yet to switch from user passwords to access tokens. Access tokens can have limited scopes, like read-only permissions.

The note doesn't need to be in these docs. I removed it.

Reply:
fair enough.. I simply meant if we are to mention here in the docs is probably deserves a bit more elaboration, removing is fine.